dataprofiler/profilers/profile_builder.py

```diff
@@ -997,12 +1001,16 @@ def _clean_data_and_get_base_stats(data, sample_size,
         len_data = len(data)
         if not len_data:
             return data, {
-                "sample_size": 0, "empty_line_count": dict(), "sample": [],
+                "sample_size": 0, "empty_line_count": dict(),
+                "sample": [], "capacity": 0
             }
 
         # ensure all data are of type str
         data = data.apply(str)
 
+        # get capacity
+        base_stats = {"capacity": utils.get_capacity(data)}
```

Comment:
This requires looping through the data. below we have a loop (albeit a sampled loop). Do we want to just put get_capacity in the for loop so we don't have to do multiple passes? 

I guess we have to determine if capacity refers to the file size or the data processed size.

Reply:
imo, we should calculate the capacity for the whole input, not the sampled one, just like we did with total_sample. I'm open to a discussion on it though.